lib/ecto/query/planner.ex

```diff
@@ -434,60 +434,69 @@ defmodule Ecto.Query.Planner do
 
   defp plan_joins([%JoinExpr{assoc: {ix, assoc}, qual: qual, on: on, prefix: prefix} = join|t],
                      query, joins, sources, tail_sources, counter, offset, adapter) do
-    schema = schema_for_association_join!(query, join, Enum.fetch!(Enum.reverse(sources), ix))
-    refl = schema.__schema__(:association, assoc)
 
```

Comment:
Empty line after function header

Reply:
I guess `mix format` should remove it right?

Reply:
We are not formatting Ecto files though, yet.